### PR TITLE
qtdeclarative: remove deprecated qmlbundle

### PR DESCRIPTION
**Pull request by GordanM**
Signed-of-by: Gordan Markuš <gordan.markus@pelagicore.com>

### CODE REVIEW

File: layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend [hunk 1]
```diff
@@ -1,10 +1,9 @@
 #
-#   Copyright (C) 2015 Pelagicore AB
+#   Copyright (C) 2017 Pelagicore AB
```
Comment (rpannek) [inline layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend]: Shouldn't this be 2015-2017?

File: layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 #
 inherit update-alternatives
 
-ALTERNATIVE_qtdeclarative-tools = "qmlbundle qmlmin qmlscene qmltestrunner"
-ALTERNATIVE_TARGET[qmlbundle] = "${OE_QMAKE_PATH_BINS}/qmlbundle"
```
Comment (rpannek) [inline layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend]: Any info on why qmlbundle is not needed anymore? Is it done in some different layer?

Comment (GordanM) [inline layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend]: Qmlbundle is a tool that allows combining resources, like QML and JavaScript files and images, into a single file. The current implementation is a prototype and is not mentioned in the Qt documentation. Developers have stopped working on it, a decision was made to remove qmlbundle. As of Qt 5.9 qmlbundle is no longer a part of qtdeclarative.

File: layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend (hunk 1 above)
Comment (DunderRoffe) [inline layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend]: yes it should be like that

File: layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend (hunk 2 above)
Comment (rpannek) [inline layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend]: Ok, great, could you put this description into the commit message so when I'm reading the commit log in the future I can read about the reason behind removing it there?

Comment (GordanM) [inline layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend]: Sure will do @rpannek. The deprecated word kinda summarizes it though.

Comment (rpannek) [inline layers/qt5-layer/recipes-qt/qt5/qtdeclarative_%.bbappend]: It was not clear to mi if de depricated it or the upstream project or for what reason, it's good to have it in the headline though.

**Pull request merged**